Source/CesiumRuntime/Private/Tests/CesiumFeatureIdSet.spec.cpp

```diff
@@ -542,7 +542,8 @@ void FCesiumFeatureIdSetSpec::Define() {
       FHitResult Hit;
       Hit.Component = pPrimitiveComponent;
       Hit.FaceIndex = 0;
-      Hit.Location = FVector_NetQuantize(0, -1, 0);
+      Hit.Location = FVector_NetQuantize(0, -1, 0) *
```

Comment:
Nitpick here, the 0 components are unnecessarily getting multiplied by the scale factor